### PR TITLE
Prims Algorithm in C#

### LINKED ISSUE #2201: Prims Algorithm
- [x] C
- [x] C++
- [ ] C# - @ritish099
- [ ] CoffeeScript
- [ ] Dart - @Swarnimashukla
- [ ] Go - @adi44
- [x] Java
- [x] JS - @sukritishah15
- [x] Kotlin - @darkmatter18
- [ ] Php - @prince-09
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript - @BhuwanChandra

### LINKED ISSUE #2206: Johnson Algorithm
- [x] C 
- [ ] C++ - @Aakansha99
- [x] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @BhardwajShweta25
- [ ] Java - @Neelaksh-Singh
- [ ] JS - @thakran14
- [x] Kotlin - @ritish099
- [x] Php - @ritish099
- [ ] Python - @BishalKrChanda
- [ ] Readme - @AdityaSingh17
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #2201

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Prims Algorithm 
- 

#### Languages Used:
- C#
- 

#### Files Added:
- Primis_Algorithm.css
- 

Thanks!

### CODE REVIEW

**Comment by ritish099** [comment]
Mam @asha15 , Please Check it out

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
+﻿using System;
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:1]: keep a line space

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+namespace Declaring_Method
+{
+    public class Prim_Algorithm
+    {
+        public static void Main(string[] args)
+        {
+            int a=0, b=0, u=0, v=0, n, i, j, edge_no = 1;
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:8]: please keep space around =

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            int min, mincost = 0;
+            int[] visited = new int[100];
+            int[,] cost = new int[10,10];
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:11]: keep a line space after this 
keep space after comma

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            Console.WriteLine("Enter the number of Nodes");
+            n = Convert.ToInt32(Console.ReadLine()); ;
+            Console.WriteLine("Enter the adjency matrix");
+
+            //Intilize all visited elements to 0
+            for(i=0;i<100;i++)
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:17]: keep space around operators

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    public class Prim_Algorithm
+    {
+        public static void Main(string[] args)
+        {
+            int a=0, b=0, u=0, v=0, n, i, j, edge_no = 1;
+            int min, mincost = 0;
+            int[] visited = new int[100];
+            int[,] cost = new int[10,10];
+            Console.WriteLine("Enter the number of Nodes");
+            n = Convert.ToInt32(Console.ReadLine()); ;
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:13]: remove the extra semi colon

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+            Console.WriteLine("Enter the adjency matrix");
+
+            //Intilize all visited elements to 0
+            for(i=0;i<100;i++)
+            {
+                visited[i] = 0;
+            }
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:20]: keep a line space after this

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+            //Taking Matrix Input from the User
+            for (i = 1; i <= n; i++)
+            {
+                for (j = 1; j <= n; j++)
+                {
+                    cost[i,j] = Convert.ToInt32(Console.ReadLine());
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:26]: keep a space after comma

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+                    //Initialize Cost Matrix to Infintiy if user input is 0
+                    if (cost[i,j] == 0)
+                        cost[i,j] = 999;
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:29]: keep a space after comma

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+                }
+            }
+            visited[1] = 1;
+            Console.WriteLine(" ");
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:33]: keep a line space after this

File: Prims_Algorithm/Prims_Algorithm.cs [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+                    Console.WriteLine("Edge {0} : ({1} {2}) cost:{3} ", edge_no++, a, b, min);
+                    mincost += min; // Calculating minimum Cost
+                    visited[b] = 1;
+                }
+                cost[a,b] = cost[b,a] = 999;
+            }
+            Console.WriteLine("Minimum Cost : {0}", mincost);
+        }
+    }
+}
```
Comment (asha15) [inline Prims_Algorithm/Prims_Algorithm.cs:58]: keep a line space after this

**Comment by asha15** [comment]
please squash your commits and let me know

**Comment by ritish099** [comment]
Mam @asha15 , Check it out

**Pull request merged**